src/adapters/libuv.h

```diff
@@ -191,7 +191,7 @@ uvAsyncAttach(natsLibuvEvents *nle)
         s = NATS_NO_MEMORY;
 
     if ((s == NATS_OK)
-        && (uv_poll_init_socket(nle->loop, nle->handle, nle->socket) != 0))
```

Comment:
natsLibuvEvents.socket is an int, and the third argument to uv_poll_init_socket is a uv_os_sock_t. On Unices uv_os_sock_t is an int, but on Windows it is a SOCKET. So this might not compile on Windows (don't have a machine handy to try out, sorry).